### DISCUSSION TITLE
【问题反馈】点击免责声明对话框外侧会退出应用

### DISCUSSION THREAD

**Discussion by FrzMtrsprt** [xiaoyaocz/flutter_dmzj#86] [Q&A] [answered]
### 问题
进入应用后，在“我的”页面点击免责声明，然后点击对话框外侧，等效于点击“退出”按钮，应用会直接退出。

### 期望行为
方案 1：使免责声明对话框无法通过点击对话框外侧解除。
方案 2：通过“我的”页面打开免责声明对话框时，点击对话框外侧等效于“同意”按钮，而非“退出”按钮。

**Comment by xiaoyaocz** [comment, marked as answer]
#95
